src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const lowDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
```

Comment:
It's a better practice to store these values as constant, improving the readability and making it easier to change them